code/game/objects/items/stacks/sheets/sheet_types.dm

```diff
@@ -809,6 +810,7 @@ new /datum/stack_recipe("paper frame door", /obj/structure/mineral_door/paperfra
 /obj/item/stack/sheet/cotton/durathread
 	name = "raw durathread bundle"
 	desc = "A bundle of raw durathread ready to be spun on the loom."
+	max_amount = 80
```

Comment:
no in-game impact but durathread doesn't need the max_amount variable, it inherits max_amount = 80 from /obj/item/stack/sheet/cotton

Reply:
Aware its their for clarity. Like how some times you see extra vars/defines on child versos of other mats.

Reply:
That's not a good reason though, considering items, stacks and sheets have dozens and dozens variables and restating them is a waste of lines.